configs/defaults/gatk_sv_multisample_1.toml

```diff
@@ -0,0 +1,29 @@
+[workflow]
+name = 'gatk_sv'
+dataset_gcp_project = 'seqr-308602'
+dataset = 'seqr'
+ref_fasta = 'gs://cpg-common-main/references/hg38/v0/dragen_reference/Homo_sapiens_assembly38_masked.fasta'
+
+[images]
+cpg_workflows = "australia-southeast1-docker.pkg.dev/cpg-common/images/cpg_workflows:pr-363"
+
+# GATK-SV specific
+cloud_sdk_docker = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv/cloud-sdk"
+cnmops_docker = "australia-southeast1-docker.pkg.dev/cpg-common/images/cnmops:v0.27.0"
+condense_counts_docker = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv/gatk:4.2.6.1-57-g9e03432"
+gatk_docker = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv/gatk:4.2.6.1-57-g9e03432"
+genomes_in_the_cloud_docker = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv/genomes-in-the-cloud:2.3.2-1510681135"
+linux_docker = "australia-southeast1-docker.pkg.dev/cpg-common/images/ubuntu1804"
+samtools_cloud_docker = "australia-southeast1-docker.pkg.dev/cpg-common/images/samtools-cloud:v0.27.0"
+sv_base_docker = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv-base:v0.27.0"
+sv_pipeline_docker = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv-pipeline:v0.27.0"
+sv_base_mini_docker = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv-base-mini:v0.27.0"
+
+# variant callers
+manta_docker = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv/manta:5994670"
+scramble_docker = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv/scramble:1.0.2"
+wham_docker = "australia-southeast1-docker.pkg.dev/cpg-common/images/sv/wham:5994670"
+
+[sv_ref_panel]
```

Comment:
I assume params like this one are consistent across all the new gatk_sv configs. Wondering if there's a better way to not repeat this? Perhaps we can leverage that generic gatk_sv.toml if not already.

Reply:
I've pushed something sketchy for this. It's a bummer at the moment splitting this across different 'workflows', but eventually all this will be replaced with just two workflows, at which point all the silly configs will be deleted